src/submission/Nav.jsx

```diff
@@ -37,8 +41,9 @@ export default class EditsNav extends Component {
           this.props.editsFetched && this.navMap.upload.isCompleted(),
         isErrored: () => this.props.code === SYNTACTICAL_VALIDITY_EDITS,
         isCompleted: () =>
-          this.navMap['syntactical & validity edits'].isReachable() &&
-          this.props.code > SYNTACTICAL_VALIDITY_EDITS,
+          (this.navMap['syntactical & validity edits'].isReachable() &&
+            this.props.code > SYNTACTICAL_VALIDITY_EDITS) ||
+          this.props.code === NO_SYNTACTICAL_VALIDITY_EDITS,
```

Comment:
Adding the `NO_` status here (and below) but not sure it's necessary. Seeing it run completely in certain cases will help.